sky/serve/autoscalers.py

```diff
@@ -74,16 +74,10 @@ def __init__(self, service_name: str,
                                   is not None else spec.min_replicas)
         # Target number of replicas is initialized to min replicas
         self.target_num_replicas: int = spec.min_replicas
-        self.latest_version: int = constants.INITIAL_VERSION
         self.update_mode = serve_utils.DEFAULT_UPDATE_MODE
 
-    def update_version(self, version: int, spec: 'service_spec.SkyServiceSpec',
+    def update_version(self, spec: 'service_spec.SkyServiceSpec',
                        update_mode: serve_utils.UpdateMode) -> None:
-        if version <= self.latest_version:
-            logger.error(f'Invalid version: {version}, '
-                         f'latest version: {self.latest_version}')
-            return
```

Comment:
We should still keep this behavior to avoid concurrent `sky update`. We should only adopt the latest update.